src/components/Post.js

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import axiosInstance from "../api/axios";
+import img1 from "../images/img1.jpg";
+import { useNavigate } from "react-router-dom";
+
+const Post = () => {
+  const [blog, setBlog] = useState({ title: "", content: "", author: "" });
+  const [postErr, setPostErr] = useState("")
+
+
+  const handleBlogDetails = (e) => {
+    e.preventDefault();
+    setBlog((current) => ({ ...current, [e.target.name]: e.target.value }));
+  };
+
+  let navigate = useNavigate();
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const result = await axiosInstance.get("/", blog, {
+
+      });
+      console.log(result)
+      setBlog();
+      navigate("/blogs");
+    } catch (e) {
+      console.log(e);
+      setPostErr("There was a problem sending post")
+    }
+  };
+
+  return (
+    <>
+      <form className="post-conatiner" onSubmit={handleSubmit}>
+        <div className="post-image-container">
+          <h3 className="post-header-text">Blog Here...</h3>
+          <div>{postErr}</div>
```

Comment:
Shouldn't this be conditionally rendered?

Reply:
Yes it must be only authenticated user must be able to post.